react/features/toolbox/components/web/Toolbox.js

```diff
@@ -989,6 +990,9 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
+            this._isModerator() && <LiveStreamButton
```

Comment:
Maybe better is merge develop and nexy use _isModerator prop form the my request that you looked at morning?

Reply:
In any case, please use `_isModerator` prop.

Reply:
Fixed